### PR TITLE
New package: mmv-go 0.1.6

### PR DESCRIPTION
**Pull request by akierig**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl (crossbuild)
  - aarch64 (crossbuild)
  - aarch64-musl (crossbuild)

### CODE REVIEW

File: srcpkgs/mmv-go/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'mmv-go'
+pkgname=mmv-go
+version=0.1.6
+revision=1
+build_style=go
+go_import_path=github.com/itchyny/mmv/cmd/mmv@latest
+short_desc="Rename and move multiple files at once with your editor"
+maintainer="anelki <akierig@fastmail.de>"
+license="MIT"
+homepage="https://github.com/itchyny/mmv"
```
Comment (mhmdanas) [inline srcpkgs/mmv-go/template:10]:
```suggestion
homepage="https://github.com/itchyny/mmv"
changelog="https://raw.githubusercontent.com/itchyny/mmv/main/CHANGELOG.md"
```

File: srcpkgs/mmv-go/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+distfiles="https://github.com/itchyny/mmv/archive/refs/tags/v${version}.tar.gz"
+checksum=18d181e6a53437ad18c4409f3c423bc23687152a17b53a605dde13dfffb0d215
+
+post_install() {
+	mv ${DESTDIR}/usr/bin/mmv ${DESTDIR}/usr/bin/mmv-go
```
Comment (mhmdanas) [inline srcpkgs/mmv-go/template:15]:
```suggestion
	mv ${DESTDIR}/usr/bin/mmv{,-go}
```
or, if you prefer,
```suggestion
	mv ${DESTDIR}/usr/bin/{mmv,mmv-go}
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: srcpkgs/mmv-go/template (hunk 2 above)
Comment (paper42) [reply srcpkgs/mmv-go/template:15]: I don't think that's necessary or more readable

Comment (mhmdanas) [reply srcpkgs/mmv-go/template:16]: I prefer it for brevity, but it's no big deal.

**Pull request merged**